ext/opcache/config.m4

```diff
@@ -42,6 +42,10 @@ if test "$PHP_OPCACHE" != "no"; then
         PHP_OPCACHE_JIT=no
         ;;
     esac
+    if test "$host_vendor" = "apple" -a "$host_cpu" = "aarch64" -a "$ZEND_ZTS" = "yes"; then
```

Comment:
```suggestion
    if test "$host_vendor" = "apple" && test "$host_cpu" = "aarch64" && test "$PHP_THREAD_SAFETY" = "yes"; then
```

The `-a` and `-o` are obsolete according to [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html) and PHP_THREAD_SAFETY was introduced in 4633e70ab1d03cbbfdb166a3c1912d2277c02882 for extensions.

Reply:
Thank you! I'm no good at autoconf. ^^